mods/ra/weapons/missiles.yaml

```diff
@@ -173,193 +124,102 @@ MammothTusk:
 			Wood: 75
 			Light: 60
 			Heavy: 25
+			Concrete: 50
 		DamageTypes: Prone50Percent, TriggerProne, SmallExplosionDeath
-	Warhead@2Smu: LeaveSmudge
-		SmudgeType: Crater
-		InvalidTargets: Vehicle, Tank, Structure, Wall, Husk, Trees
 	Warhead@3Eff: CreateEffect
-		Explosions: med_explosion
-		ImpactSounds: kaboom12.aud
 		InvalidImpactTypes: Water, Air, AirHit
-	Warhead@4EffWater: CreateEffect
-		Explosions: small_splash
-		ImpactSounds: splash9.aud
-		ValidImpactTypes: Water
 	Warhead@5EffAir: CreateEffect
 		Explosions: med_explosion_air
 		ImpactSounds: kaboom25.aud
 		ValidImpactTypes: Air, AirHit
 
 Nike:
+	Inherits: ^AntiAirMissile
```

Comment:
> This changes the deathtype from SmallExplosionDeath to ExplosionDeath. I don't think this matters in practice?

AA missiles are unlikely to damage, let alone kill infantry, so I didn't bother keeping this the way it was.